service/schema.py

```diff
@@ -0,0 +1,25 @@
+import json
+from jsonschema import validate
+
+schema = {
```

Comment:
What do we need this schema for?

Reply:
this one might be on me, I asked to be able to validate the messages being received against an expected message structure and said it didn't matter what schema for now.

Reply:
believe mike added this requirement is your absence -i do agree its helpful having some sort of validation on the message payload before it goes further down the pipe